teuthology/provision/cloud/util.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
It'd be nice to have some docstrings for these, particularly the way in which combine_dicts combines them, but in general